MediaBrowser.Api/Library/LibraryService.cs

```diff
@@ -988,19 +988,16 @@ private int GetCount(Type type, User user, GetItemCounts request)
         /// Posts the specified request.
         /// </summary>
         /// <param name="request">The request.</param>
-        public void Post(RefreshLibrary request)
+        public async Task Post(RefreshLibrary request)
         {
-            Task.Run(() =>
```

Comment:
I'm still not convinced this won't be slower

Reply:
Only 1 way to know for sure ;)